.gitignore

```diff
@@ -7,4 +7,6 @@ coverage/**
 .idea
 beaconchain
 lodestar-db
-lodestar-config.toml
\ No newline at end of file
+lodestar-config.toml
+keys/*
+!keys/validator/.gitkeep
```

Comment:
I don't think we should keep that directory in git. It should be generated depending on key output path